docs/integrations/ios-specific-workflow-steps/swift-lint.md

```diff
@@ -0,0 +1,77 @@
+---
+title: SwiftLint
+metaTitle: SwiftLint
+metaDescription: SwiftLint
+sidebar_position: 4
+---
+
+import Screenshot from '@site/src/components/Screenshot';
+
+# SwiftLint
+SwiftLint checks the source code for programmatic as well as stylistic errors. This is helpful in identifying some common and uncommon mistakes that are made during coding. This step installs [SwiftLint](https://github.com/realm/SwiftLint/) and runs SwiftLint with given options. 
+
+### Prerequisites
+
+Remember, SwiftLint must be used after the following steps in order to work as expected.
+
+| Prerequisite Workflow Step                      | Description                                     |
+|-------------------------------------------------|-------------------------------------------------|
+| [Git Clone](https://docs.appcircle.io/workflows/common-workflow-steps/#git-clone) | The repo needs to be cloned in order to start the Cocoapods installation process. After the clone, Cocoapods will be installed and then SwiftLint step will be run. After this step works, the variable `AC_REPOSITORY_DIR` will be created. This variable is the input variable for Cocoapods and SwiftLint. |
+| [Cocoapods Install](https://docs.appcircle.io/workflows/ios-specific-workflow-steps#cocoapods-install)| This step will install the dependencies in the project before SwiftLint can run. |
+
+<Screenshot url='https://cdn.appcircle.io/docs/assets/BE2613-lint_order.png' />
+
+:::warning
+If you are using **Cocoapods**, note that this step is dependent on the [**Cocoapods Install**](https://docs.appcircle.io/workflows/ios-specific-workflow-steps#cocoapods-install) step. Otherwise the SwiftLint component will **fail** and the **pipeline will break.**
+:::
+:::caution
+If you are using **Swift Package Manager(SPM)**, do not use this step. SPM packages will be compiled in other steps that work with **Xcode**, such as [**Xcodebuild for Devices**](https://docs.appcircle.io/workflows/ios-specific-workflow-steps#xcodebuild-for-devices-archive--export)
+
+If you have SPM in your project and you are using SwiftLint component in workflow. The Linter component will give an error because it cannot find the required dependencies.
+:::
+
+### Step Details
+
+You can customize your SwiftLint step with the options in the component. You can also customize your SwiftLint step with the options in the component. You can also find detailed descriptions of all variables in the table below.
+
+- With the Range option, you can run SwiftLint on your entire project or only on changes in PR.
+
+<Screenshot url='https://cdn.appcircle.io/docs/assets/BE2613-lint_range.png' />
+
+- You can change the report type with the Reporter format option. This option supports extensions such as **`html`**, **`json`**, **`junit`** etc.
```

Comment:
@boztopuz Can you add this description to the table on the `AC_LINT_REPORTER` row and after that remove it from there?

In this way, the document would be more understandable as we would avoid repetitions and collect the same information in one place.